hathor/transaction/storage/rocksdb_storage.py

```diff
@@ -108,20 +108,6 @@ def _import_from_other(self, other_storage: TransactionStorage) -> None:
             # handled by the manager after pre_init
             self._save_transaction(tx)
 
-    def pre_init(self) -> None:
-        from rocksdb.errors import RocksIOError
-        if self.is_empty():
-            # try to load the old rocksdb storage
-            from hathor.transaction.storage.old_rocksdb_storage import TransactionOldRocksDBStorage
-            try:
-                old_storage = TransactionOldRocksDBStorage(self._path)
-            except RocksIOError:
-                # old storage does not exist, no need to continue
-                return
-            self._import_from_other(old_storage)
```

Comment:
Should we remove the method `_import_from_other(...)` as well?

Reply:
Yes!

Reply:
Done.